src/gantt.ts

```diff
@@ -26,16 +26,18 @@
 
 import "./../style/gantt.less";
 
-import {select as d3Select, Selection as d3Selection} from "d3-selection";
-import {ScaleTime as timeScale} from "d3-scale";
+import "d3-transition";
+import { select as d3Select, Selection as d3Selection } from "d3-selection";
+import { ScaleTime as timeScale } from "d3-scale";
 import {
     timeDay as d3TimeDay,
     timeHour as d3TimeHour,
     timeMinute as d3TimeMinute,
     timeSecond as d3TimeSecond
 } from "d3-time";
-import {nest as d3Nest} from "d3-collection";
-import "d3-transition";
+import { nest as d3Nest } from "d3-collection";
+import { drag as d3Drag, D3DragEvent, SubjectPosition } from "d3-drag";
```

Comment:
better to import methods from libraries with a name representation to this library 
`SubjectPosition as d3SubjectPosition` same as you do for some other imports. Please check and do the same for all d3 methods (e.g. change timeScale to d3ScaleTime on the line 31 above).